pallets/encrypted-transactions/src/lib.rs

```diff
@@ -50,7 +55,7 @@ pub mod ecdsa {
 
 pub trait Trait: frame_system::Trait + pallet_session::Trait {
     type Event: From<Event<Self>> + Into<<Self as frame_system::Trait>::Event>;
-    type Tokens: MultiTokenCurrency<Self::AccountId>;
+    type Tokens: MultiTokenCurrencyExtended<Self::AccountId>;
```

Comment:
Not sure if we need to do this to use create in mock. But we can leave it like it is for now.